### PR TITLE
Add dragon attenuation model.

### PR DESCRIPTION
**Pull request by emackey**
See the [README](https://github.com/KhronosGroup/glTF-Sample-Models/blob/6cc9746a47907b549b0fc254b5d13a4528ff56df/2.0/DragonAttenuation/README.md).

### CODE REVIEW

**Comment by emackey** [comment]
Things to consider here:

- [ ] Scale the `thicknessFactor` from 3.5 to 2.2, and scale `attenuationDistance` by the same percentage.  This would get the `thicknessFactor` to more closely match the raw vertex distances at the thickest portion of the dragon's body.
- [ ] Change the thickness texturemap here to only use the green channel, to make sure implementations are reading from the correct channel.

**Comment by emackey** [comment]
Actually I'm not sure 2.2 is the thickest part, it looks like some lower parts are above 3.0.  Would be interesting to compare to the path tracer if they add support for attenuation there.

**Comment by emackey** [comment]
Also:

- [ ] Add "common problems" section with a screenshot of what happens when the node scaling is not accounted for.

**Comment by emackey** [comment]
This is ready for review.

File: 2.0/README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Models demonstrating features of extensions to the glTF specification. See glTF
 
 | Model                              | Screenshot                                            | Description |
 |------------------------------------|-------------------------------------------------------|-------------|
+| [Dragon Attenuation](DragonAttenuation) | ![](DragonAttenuation/screenshot/screenshot.jpg) | Dragon with background, using [material variants](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_variants), [transmission](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_transmission), and volume. |
```
Comment (abwood) [inline 2.0/README.md:110]:
```suggestion
| [Dragon Attenuation](DragonAttenuation) | ![](DragonAttenuation/screenshot/screenshot.jpg) | Dragon with background, using [material variants](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_variants), [transmission](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_transmission), and [volume](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_volume). |
```

Comment (emackey) [reply 2.0/README.md]: Thanks!

**Review by abwood — APPROVED** [review]
Great model and fantastic writeup. Thanks @emackey!

**Pull request merged**